proptest/src/test_runner/runner.rs

```diff
@@ -231,7 +231,10 @@ impl TestRunner {
         test: impl Fn(S::Value) -> TestCaseResult,
     ) -> TestRunResult<S> {
         let tree = strategy.new_tree(self).unwrap();
-        test(tree.current()).unwrap();
+        assert!(matches!(
+            test(tree.current()),
+            Ok(_) | Err(TestCaseError::Reject(_))
```

Comment:
can you please add a comment here? I like this explanation: https://altsysrq.github.io/rustdoc/proptest/latest/proptest/test_runner/enum.TestCaseError.html#variant.Reject

Reply:
[1a12977](https://github.com/model-checking/kani/pull/2554/commits/1a12977f4df05ddd12c00fbae36c9ac298ccd1b0)